### PR TITLE
Make Body Fragment, fix some things

### PR DESCRIPTION
**Pull request by pulsejet**

### CODE REVIEW

File: app/src/main/java/in/ac/iitb/gymkhana/iitbapp/fragment/EventFragment.java [hunk 1]
```diff
@@ -148,52 +149,52 @@ public void onClick(View view) {
...
+                final int endStatus = event.getEventUserUes() == status ? 0 : status;
+                RetrofitInterface retrofitInterface = ServiceGenerator.createService(RetrofitInterface.class);
+                retrofitInterface.updateUserEventStatus(((MainActivity) getActivity()).getSessionIDHeader(), event.getEventID(), endStatus).enqueue(new Callback<Void>() {
+                    @Override
+                    public void onResponse(Call<Void> call, Response<Void> response) {
+                        if (response.isSuccessful()) {
+                            event.setEventUserUes(endStatus);
+                            new updateDbEvent().execute(event);
+                            interestedButton.setBackgroundColor(getResources().getColor(endStatus == 1 ? R.color.colorAccent : R.color.colorWhite));
+                            goingButton.setBackgroundColor(getResources().getColor(endStatus == 2 ? R.color.colorAccent : R.color.colorWhite));
```
Comment (unstablebrainiac) [inline app/src/main/java/in/ac/iitb/gymkhana/iitbapp/fragment/EventFragment.java]: Replace numbers with Constants.STATUS_*

File: app/src/main/res/layout/fragment_event_card.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (unstablebrainiac) [inline app/src/main/res/layout/fragment_event_card.xml]: Are you looking for feed_card.xml?

File: app/build.gradle [hunk 3]
```diff
@@ -62,5 +62,6 @@ dependencies {
     implementation "com.android.support:cardview-v7:${supportLibVersion}"
     implementation "de.hdodenhof:circleimageview:${circleImageViewVersion}"
     implementation "ru.noties:markwon:${markwonVersion}"
+    compile 'com.jakewharton.picasso:picasso2-okhttp3-downloader:1.1.0'
```
Comment (unstablebrainiac) [inline app/build.gradle]: Replace compile with implementation

File: app/src/main/java/in/ac/iitb/gymkhana/iitbapp/adapter/EventAdapter.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
+package in.ac.iitb.gymkhana.iitbapp.adapter;
```
Comment (unstablebrainiac) [inline app/src/main/java/in/ac/iitb/gymkhana/iitbapp/adapter/EventAdapter.java]: Are you looking for FeedAdapter.java?

File: app/src/main/java/in/ac/iitb/gymkhana/iitbapp/data/Body.java [hunk 5]
```diff
@@ -49,8 +49,11 @@
     @ColumnInfo(name = "blog_url")
     @SerializedName("blog_url")
     String bodyBlogURL;
+    @ColumnInfo(name = "user_follows")
+    @SerializedName("user_follows")
+    Boolean bodyUserFollows;
```
Comment (unstablebrainiac) [inline app/src/main/java/in/ac/iitb/gymkhana/iitbapp/data/Body.java]: Replace Boolean with boolean

File: app/src/main/java/in/ac/iitb/gymkhana/iitbapp/fragment/EventFragment.java (hunk 1 above)
Comment (pulsejet) [reply app/src/main/java/in/ac/iitb/gymkhana/iitbapp/fragment/EventFragment.java]: +1

File: app/src/main/res/layout/fragment_event_card.xml (hunk 2 above)
Comment (pulsejet) [reply app/src/main/res/layout/fragment_event_card.xml]: Yeah .... *cry*

File: app/src/main/java/in/ac/iitb/gymkhana/iitbapp/data/Body.java (hunk 5 above)
Comment (pulsejet) [reply app/src/main/java/in/ac/iitb/gymkhana/iitbapp/data/Body.java]: Done

File: app/build.gradle (hunk 3 above)
Comment (pulsejet) [reply app/build.gradle]: Done

**Pull request merged**